src/package/nuspec/TestPlatform.Build.nuspec

```diff
@@ -14,7 +14,7 @@
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
   </metadata>
   <files>
-    <file src="Microsoft.TestPlatform.Build\netstandard1.3\Microsoft.TestPlatform.Build.dll" target="lib\netstandard1.3\" />
-    <file src="Microsoft.TestPlatform.Build\netstandard1.3\Microsoft.TestPlatform.targets" target="runtimes\any\native" />
+    <file src="Microsoft.TestPlatform.Build\Debug\netstandard1.3\Microsoft.TestPlatform.Build.dll" target="lib\netstandard1.3\" />
```

Comment:
yes, this will break official nuget packages. Why is this required? I tried to mimic the steps we're doing (create artifacts\Debug\MS.TP.Build\netstandard1.3 directory structure, copy MS.TP.Build.csproj to artifacts\Debug, then call dotnet pack; it seemed to package binaries without this change).